tox.ini

```diff
@@ -1,8 +1,74 @@
 [tox]
-envlist = py26,py27,py33,py34,py35,pypy
+envlist =
+    py26
+    py27
+    py33
+    py34
+    py35
+    py26-no-yaml
+    py27-no-yaml
+    py33-no-yaml
+    py34-no-yaml
+    py35-no-yaml
+    pypy-no-yaml
+    lint
 
 [testenv]
 deps=
-	pytest	
+    pytest
+    PyYAML
+    coverage
 commands=
-	py.test tests munch --doctest-modules
+    coverage run --source=munch -m py.test
+
+
+[testenv:py26-no-yaml]
+deps=
```

Comment:
This can be solved with a single `[testenv]` directive, as seen here http://tox.readthedocs.org/en/latest/config.html#generating-environments-conditional-settings